core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/add-answer-group-modal.template.html

```diff
@@ -81,6 +81,10 @@ <h3>Add Response</h3>
   .add-answer-group-modal .oppia-nothing-text {
     color: #888;
   }
+  .md-container .md-ink-ripple {
```

Comment:
This is not required in this PR, right? We need to run the job first and then decide what to do after looking at the output.